package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "scripts": {
     "prepare": "husky install",
-    "install": "lerna clean && lerna init --hoist",
```

Comment:
@ramboz some how the `--hoist` option is failing when I'm running `npm run build`, so I've removed it.

Reply:
Yes, this is an old lerna instrumentation but the new version breaks this it seems